app/Models/User.php

```diff
@@ -2,36 +2,27 @@
 
 namespace Rogue\Models;
 
-use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Auth\Authenticatable;
+use Illuminate\Contracts\Auth\Authenticatable as AuthenticatableContract;
+use DoSomething\Gateway\Contracts\NorthstarUserContract;
+use DoSomething\Gateway\Laravel\HasNorthstarToken;
 
-class User extends Authenticatable
+class User extends Model implements AuthenticatableContract, NorthstarUserContract
 {
-    /**
-     * The attributes that are mass assignable.
-     *
-     * @var array
-     */
-    protected $fillable = [
```

Comment:
I think it makes sense to keep this property defined as an empty array to protect against mass-assignment attacks if we ever exposed this model to form input somehow.